### PR TITLE
chore(sdk): ensure AWS SDK deps are in sync

### PR DESCRIPTION
**Pull request by garysassano**
Making it easier to keep deps in sync for AWS SDK clients.

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Comment by tsuf239** [comment]
Hi Gary, would you like to proceed with this PR? 
(if so you'll have to resolve the conflicts and we can take a look to see why the build was failing)

**Comment by garysassano** [comment]
@tsuf239 I resolved the conflicts, there's just one issue that I couldn't figure out.

There's a `bundled 0.0.8` dependency that gets added somehow.

```sh
dependencies:
- @aws-sdk/client-cloudwatch-logs 3.449.0
+ @aws-sdk/client-cloudwatch-logs 3.511.0
- @aws-sdk/client-dynamodb 3.449.0
+ @aws-sdk/client-dynamodb 3.511.0
- @aws-sdk/client-elasticache 3.449.0
+ @aws-sdk/client-elasticache 3.511.0
- @aws-sdk/client-lambda 3.449.0
+ @aws-sdk/client-lambda 3.511.0
- @aws-sdk/client-s3 3.449.0
+ @aws-sdk/client-s3 3.511.0
- @aws-sdk/client-secrets-manager 3.449.0
+ @aws-sdk/client-secrets-manager 3.511.0
- @aws-sdk/client-sns 3.449.0
+ @aws-sdk/client-sns 3.511.0
- @aws-sdk/client-sqs 3.449.0
+ @aws-sdk/client-sqs 3.511.0
- @aws-sdk/s3-request-presigner 3.449.0
+ @aws-sdk/s3-request-presigner 3.511.0
- @aws-sdk/util-dynamodb 3.449.0
+ @aws-sdk/util-dynamodb 3.511.0
+ bundled 0.0.8
```

**Comment by tsuf239** [comment]
yes, I'm not sure I know why, maybe because you add them both in `bundledDeps` and then in `addDeps`, is it gone if you try doing `@aws-sdk/util-dynamodb${AWS_SDK_VERSION}` (like `cdktf@${CDKTF_VERSION}` in line 62) for each, and remove lines 153-158?

**Comment by garysassano** [comment]
Ok, there was actually a dedicated method to specify that deps need to be bundled. Now it looks all good.

File: libs/wingsdk/.projen/deps.json [hunk 1]
```diff
@@ -167,57 +167,56 @@
...
     },
     {
       "name": "@aws-sdk/s3-request-presigner",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/types",
-      "version": "3.449.0",
```
Comment (tsuf239) [inline libs/wingsdk/.projen/deps.json:215]: why do you skip this?

Comment (garysassano) [reply libs/wingsdk/.projen/deps.json:215]: It's in the notes: https://github.com/winglang/wing/pull/5429/files#diff-0769c0418cbdd805ac163690fe215884e186a895522b0e382e80b992a0eb3487R150-R151

Comment (garysassano) [reply libs/wingsdk/.projen/deps.json:215]: It's in the comment: https://github.com/winglang/wing/pull/5429/files#diff-0769c0418cbdd805ac163690fe215884e186a895522b0e382e80b992a0eb3487R150-R151

**Review by MarkMcCulloh — APPROVED** [review]
Successful AWS test run https://github.com/winglang/wing/actions/runs/7994497893/job/21832694173

**Pull request merged**